library/L1_Peripheral/stm32f10x/system_controller.hpp

```diff
@@ -0,0 +1,46 @@
+// system clock class set clock outputs frequency and selects clock source

```

Comment:
Delete this line here and move to become the comment for the system controller. See the comment below.